packages/components/src/components/pagination/pagination-styles.ts

```diff
@@ -50,15 +64,27 @@ export const getComponentCss = (
         whiteSpace: 'nowrap',
         cursor: 'pointer',
         color: primaryColor,
-        ...getFocusJssStyle({ color: focusColor, offset: 1 }),
+        outline: 0,
```

Comment:
Why?

Reply:
to avoid the default focus outline on span